resources/user-agents/browsers/safari/safari-generic.json

```diff
@@ -257,6 +250,146 @@
           ]
         }
       ]
+    },
+    {
+      "userAgent": "Safari Generic Older",
```

Comment:
I did look around a bit to see what Safari 1.x and early 2.x useragents looked like and found these examples, so added them here as special cases.  I thought about creating a `safari-1-x.json` file, but then found some patterns for early 2.x versions of Safari that needed a home as well.